homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -30,12 +32,13 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     coordinator = hass.data[DOMAIN][config_entry.entry_id][COORDINATOR]
 
     entities = []
+    is_thermostat = api.single_master_thermostat()
```

Comment:
does this do any io or requests?

Reply:
No, all IO is handled through the updates with DuC

Reply:
No, `api` is residing in memory. It is being updated via the DUC.

Reply:
ok was mainly checking if it needed to move into executor and be awaited. Doesnt seem so based on your comments.

Reply:
It doesn't but good catch for if it did. We moved everything into DuC and only do IO in updates and (obviously) connect. Sharp eye!